infrastructure/README.md

```diff
@@ -27,8 +27,8 @@ Once the ACR is deployed, follow these manual steps to set up CD pipeline:
 
 1. Create a new variable group in Azure Pipeline Library
 2. Create variable ACR_SERVER and set value to the server name, which will be the output of your deployment (<application name>container.azurecr.io)
-3. Get values of username and password from container using Azure Portal
-4. Create variables ACR_USERNAME and ACR_PASSWORD and set them to the values you got from the Azure Portal.
+3. Get values of username and password from Key Vault deployed in global resources using Azure Portal
```

Comment:
Get username and password values from the Key Vault deployed in "global resources" resource group using the Azure Portal.

Reply:
@code4clouds: Integrated review comments.